packaging/debian/rules

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+export DH_VERBOSE = 1
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+%:
+	dh $@
+
+override_dh_auto_configure:
+	dh_auto_configure -- -DSKIP_TEST=TRUE \
+	                     -DCMAKE_C_COMPILER=clang \
```

Comment:
Why not gcc? I think clang is less popular for non-programmers.

Reply:
OK, switched to gcc.
Initially, this configuration is required when we use `-fsanitize=fuzzer`.